models/myPageDao.js

```diff
@@ -0,0 +1,26 @@
+const dataSource = require('./dataSource')
+
+const getLike = async (userId) => {
```

Comment:
```suggestion
const getLikeByUserId = async (userId) => {
```
네이밍을 다음과 같이 좀더 정확하게 작성할 수 있을 것 같아요.
좋아요 수를 get하는 다른 함수들이 생길 확률도 생각해보면 좋을 것 같아요. 가독성도 올릴 수 있구요!